src/Payum/Action/ConvertPaymentAction.php

```diff
@@ -24,6 +24,26 @@ public function execute($request): void
 
         /** @var PaymentInterface $payment */
         $payment = $request->getSource();
+        $details = $payment->getDetails();
+
+        if ([] === $details) {
```

Comment:
If you choose EveryPay at first, meaning the first time you try to pay an order, the value of the `details` will be equal to `[]`.
If the first attempt is done with another gateway like the offline one the `details` will be already filled up with something. It's an issue inside Sylius making this mess...

To be sure it won't happen, you can check for an empty array at first, and add another condition checking for the absence of a specific EveryPay field (if there is one).

This can also be done inside a Payum Extension like this one :
https://github.com/kiboko-labs/payzen-bundle/blob/master/src/Extension/FromForeignGatewayExtension.php